WordPress/wordcamp.org
# Speaker Feedback: Generate stat reports

This adds functionality within the Speaker Feedback Tool plugin for generating stats for an individual site, as well as a new report in the WordCamp Reports plugin for aggregating those stats across multiple sites into a CSV file.

Fixes #344

### Screenshots

![sft-report-ui](https://user-images.githubusercontent.com/916023/84092791-4f1da280-a9ad-11ea-9185-9172779bf184.jpg)

### How to test the changes in this Pull Request:

1. It's probably easiest to test this PR on a sandbox, which has access to speaker feedback data from multiple real WordCamps.
1. Go to the [WordCamp Speaker Feedback report](https://central.wordcamp.org/wp-admin/index.php?page=wordcamp-reports&report=wordcamp-speaker-feedback) in the WordCamp Central admin.
1. Set a date range that includes recent WordCamps that used the tool. 2020-04-01 to 2020-06-08 should give you all three.
1. Download the CSV file. You can check some of the numbers in the file against the feedback counts shown in the Feedback list table for each of the individual WordCamp sites.
